package.json

```diff
@@ -25,7 +25,7 @@
   },
   "scripts": {
     "pretest": "npm run lint && npm run beautify-lint",
-    "lint": "eslint lib",
+    "lint": "eslint .",
```

Comment:
This should really be explicit be ```"lint": "eslint example index.js mocha-env-loader.js",``` 

Targeting the project root opens up the possibility of linting something we don't want to. I say that recognizing this is knit picky and you are fixing my copy paste fail (thanks).